src/hotspot/share/gc/g1/g1CollectedHeap.hpp

```diff
@@ -1207,6 +1207,15 @@ class G1CollectedHeap : public CollectedHeap {
     collection_set_iterate_increment_from(blk, NULL, worker_id);
   }
   void collection_set_iterate_increment_from(HeapRegionClosure *blk, HeapRegionClaimer* hr_claimer, uint worker_id);
+  // Iterate the part of a regions array given by the offset and length applying the given
```

Comment:
Iterate part of an array of region indexes given by offset and length, applying the given HeapRegionClosure on each region. The worker_id....